src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -35,6 +35,29 @@ static Thread* g_RuntimeInitializingThread;
 
 #endif //!DACCESS_COMPILE
 
+#ifdef HOST_WASM
+void* Thread::GetShadowStackBottom()
+{
+    return m_pShadowStackBottom;
+}
```

Comment:
It's pretty important that these get inlined and since we don't have LTO, they should be moved to `thread.inl`.

Reply:
ok, made them `FORCEINLINE`